addons/links/src/preview.test.js

```diff
@@ -17,6 +17,8 @@ jest.mock('global', () => ({
       kind: 'kind',
     })),
   },
+  window: global,
+  __STORYBOOK_LOGGER: 'warn',
```

Comment:
what is this supposed to do? below `__STORYBOOK_LOGGER` is set to `console`, which is a different type?